sr_example/scripts/sr_example/sr_left_arm_examples.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
```

Comment:
did you update the corresponding .md file to reflect those changes (right / left)?